### PR TITLE
Removing secret key from cloud foundry. Taking all commented out code…

### PR DESCRIPTION
**Pull request by nherriot**
Removing old unused settings file.
Tidy up of cloud foundry, local and default settings files.
Removing of secret keys.
Tested locally and on cloud foundry 'dev' environment.

### CODE REVIEW

File: oauth2server/proj/settings/cloud_foundry_settings.py [hunk 1]
```diff
@@ -76,22 +72,6 @@
     DB_USERNAME = credentials.get('username', '')
```
Comment (jonmorphic) [inline oauth2server/proj/settings/cloud_foundry_settings.py]: If we default the DB credentials to empty strings, could this cause a silent failure? Would it be worth failing (with Splunk output) if the keys are not found?

File: oauth2server/proj/settings/cloud_foundry_settings.py [hunk 2]
```diff
@@ -5,22 +5,23 @@
 # /manifest_develop_cloudfoundry.yml file in the root folder.
 # Parameters used to make this settings file active are: DJANGO_SETTINGS_MODULE: proj.settings.cloud_foundry_settings.
 
-import json
 import cfenv
-
 from proj.settings.default import *
 
 remote_logger = logging.getLogger('remote')
```
Comment (jonmorphic) [inline oauth2server/proj/settings/cloud_foundry_settings.py:11]: Is this logger being configured to emit JSON output?

File: oauth2server/proj/settings/default.py [hunk 3]
```diff
@@ -39,7 +38,6 @@
 # This specifies what version and application name is displayed  in the info endpoint. (i.e. http://localhost:8008/info)
 # It also tries to detect the git_info file that is injected by jenkins in cloud foundry. If it's not there is uses
 # values from the local system.
-
 APPLICATION_NAME = 'oauth2 server'
 APPLICATION_VERSION = '0.1.2'
```
Comment (jonmorphic) [inline oauth2server/proj/settings/default.py]: Does this need a version bump?

**Review by jonmorphic — COMMENTED** [review]
Generally looks straightforward, have raised a few queries inline.

**Review by andrewmil — COMMENTED** [review]
Looks pretty fine, is there any value in keeping some of the commented code? Database code in local for exmaple.

Also the current default settings to use are local, should this be set to cloudfoundry to ensure that we don't end up pushing our local keys?

File: oauth2server/proj/settings/local.py [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 from proj.settings.default import *
 
+
+# Standard instance of a logger with __name__. We are using this so that our root folder has our logger defined.
+stdlogger = logging.getLogger(__name__)
```
Comment (andrewmil) [inline oauth2server/proj/settings/local.py]: we haven't imported `logging` here

File: oauth2server/proj/settings/local.py [hunk 5]
```diff
@@ -2,6 +2,12 @@
 
 from proj.settings.default import *
```
Comment (andrewmil) [inline oauth2server/proj/settings/local.py]: I don't think you can import `proj` like this from here I don't think, this needs to be `from oauth2server.proj.settings.default import *`

Comment (andrewmil) [reply oauth2server/proj/settings/local.py]: ^ I take this one back after running the django server but I don't understand how it works :S

**Comment by jonmorphic** [comment]
Depending how you’re running, I think Django patches the PYTHONPATH and adds the parent of manage.py.

**Review by andrewmil — DISMISSED** [review]

**Review by jonmorphic — APPROVED** [review]

File: oauth2server/proj/settings/cloud_foundry_settings.py (hunk 1 above)
Comment (nherriot) [reply oauth2server/proj/settings/cloud_foundry_settings.py]: Removed empty string. If 'vcap_services' is set and there are no DB settings the micro service will fail with type errors.

File: oauth2server/proj/settings/default.py (hunk 3 above)
Comment (nherriot) [reply oauth2server/proj/settings/default.py]: Yes. Done.

**Pull request merged**